ai_chatbots/consumers.py

```diff
@@ -121,34 +121,31 @@ async def handle(self, message: str):
         # Headers are only sent after the first body event.
         # Set "more_body" to tell the interface server to not
         # finish the response yet:
-        payload = "event: ping\ndata: null\n\n"
-        await self.send_body(payload.encode("utf-8"), more_body=True)
+        await self.send_chunk("")
 
         try:
             message_text = process_message(message, agent)
 
             for chunk in agent.get_completion(message_text):
-                await self.send_event(event=chunk, more_body=True)
+                await self.send_chunk(chunk)
```

Comment:
Locally, this iteration takes many seconds, but the body text doesn't seem to actually be sent until it finishes. It is sent incrementally if an artificial `await asyncio.sleep(0.1)` is inserted.

So, something doesn't seem right here, but it behaves this way on `main`, too, so not related to the change from SSE to regular text.